src/server/utils/prepare.js

```diff
@@ -10,21 +10,43 @@ var exec = require('child_process').exec,
  * @return {Promise}
  */
 function execCordovaPrepare(projectRoot, platform) {
-    var deferred = Q.defer();
+    return retryAsync(getExecCordovaPrepareImpl(projectRoot, platform), 2);
```

Comment:
By reading the name of the parameters from the retryAsync function, the number `2` would be the number of attempts to retry. Maybe we can avoid hardcoding this value here and define a constant at module level like,
{code}
var DEFAULT_RETRY_ATTEMPTS = 2;
{code}